client/overview/index.js

```diff
@@ -71,7 +70,6 @@ const OverviewPage = () => {
 		accountStatus,
```

Comment:
I think that now accountStatus should not be passed as it's missing in the interface https://github.com/Automattic/woocommerce-payments/pull/6696/files#diff-d773390b1a89b8ea2059380cd3f5c1997f373e3c6066cc06d3cec7299f50852fR24

Reply:
![Uploading image.png…]()